entry.go

```diff
@@ -56,13 +57,13 @@ type TunaEntry struct {
 }
 
 func NewTunaEntry(service *Service, serviceInfo *ServiceInfo, config *EntryConfiguration, wallet *nkn.Wallet) (*TunaEntry, error) {
-	common, err := NewCommon(service, serviceInfo, wallet, config.DialTimeout, config.SubscriptionPrefix, config.Reverse, nil)
+	newCommon, err := NewCommon(service, serviceInfo, wallet, config.DialTimeout, config.SubscriptionPrefix, config.Reverse, nil)
```

Comment:
Why rename?

Reply:
same name with import package, got this hint from IDE